### PR TITLE
Spelling code scanning product

### PR DESCRIPTION
**Pull request by jsoref**
follow-up to https://github.com/github/codeql/pull/10743

### CODE REVIEW

File: javascript/old-change-notes/2021-07-14-querystring.md [hunk 1]
```diff
@@ -2,5 +2,5 @@ lgtm,codescanning
 * The security queries now track taint through more query string parsers.
   Affected packages are
     [qs](https://npmjs.com/package/qs),
-    [normailize-url](https://npmjs.com/package/normalize-url),
+    [normalize-url](https://npmjs.com/package/normalize-url),
```
Comment (jsoref) [inline javascript/old-change-notes/2021-07-14-querystring.md:5]: brand

**Review by tausbn — APPROVED** [review]
All of these changes look sensible to me. 👍

**Review by erik-krogh — APPROVED** [review]
JS 👍

**Comment by smowton** [comment]
Can old, released change logs be altered like this?

**Comment by aibaars** [comment]
> Can old, released change logs be altered like this?

I think it should be fine. Fixing them also prevents copy-pasting mistakes when using an old changelog entry as "inspiration" for a new one ;-)

**Review by aibaars — APPROVED** [review]

**Review by hvitved — APPROVED** [review]
C# 👍

**Review by jketema — APPROVED** [review]
The Java and Go changes look reasonable to me. Going to merge this now.

**Pull request merged**